ts/nni_manager/common/nniConfig.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+'use strict';
+
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+
+export interface CustomEnvironmentServiceConfig {
+    name: string;
+    nodeModulePath: string;
+    nodeClassName: string;
+}
+
+function readConfigFile(fileName: string): string {
+    let configPath: string;
+    if (process.platform === 'win32') {
+        configPath = path.join(process.env.APPDATA as string, 'nni', fileName);
+    } else {
+        configPath = path.join(os.homedir(), '.config/nni', fileName);
+    }
+    return fs.readFileSync(configPath).toString();
```

Comment:
This is not consistent with the logic in `nni.runtime.config`. What will happen in the case of conda?

Reply:
Ooops, I forgot.